client/src/pages/Messages/Chat/useStyles.ts

```diff
@@ -0,0 +1,133 @@
+import { Theme } from '@mui/material/styles';
+import { makeStyles } from '@mui/styles';
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    display: 'flex',
+    flexGrow: 8,
+    flexDirection: 'column',
+  },
+  chatContainer: {
+    marginLeft: 41,
+    marginRight: 41,
+    display: 'flex',
+    flexDirection: 'column',
+    flexGrow: 1,
+    justifyContent: 'space-between',
+  },
+  root1: {
+    display: 'flex',
```

Comment:
Since we have a lot of CSS within this file, some of the significant styles can be applied as props to the relevant MUI components 👍🏽